docs/using-jda/using-new-features.md

```diff
@@ -0,0 +1,71 @@
+# Using New Features
+
+When contributors are making new features, like introducing modals, 
+you might be interested in trying out or testing these features before they get released.
+
+## Using [JitPack](https://jitpack.io)
+
+Using JitPack, you can get a build of any fork of JDA, on any branch, on any commit.
+It is still highly recommended that you use a build tool such as [Gradle](https://gradle.org) or [Maven](https://maven.apache.org)
+
+### Getting the Necessary Info
+
+Let's take the [PR for the Discord modals](https://github.com/DV8FromTheWorld/JDA/pull/2024) as an example
+
+![img.png](../assets/images/github_pr.png){width="720"}
+
+Here the important part is `from Xirado:feature/modals`, you need to extract the PR author, which here is `xirado`
```

Comment:
This sentence doesn't seem to make much sense.  I think it is fine to just say something like "You need the author's name, which here is \`Xirado\`."